catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java

```diff
@@ -309,17 +315,40 @@ private static boolean anyNull(Object... args) {
   }
 
   private static void saveMetacards(CatalogFramework framework, List<Metacard> metacards) {
-    SECURITY.runAsAdmin(
+    // TODO: This code will be removed once CRUD support is completed
+    /**
+     * Purpose: Circumvents the idea of a system template to make it look like the loaded templates
+     * were created by a std. user. Additional Info: dummyUser is created (temporarily) in
+     * user.properties/user.attributes to demonstrate template creation - This sets the metacard
+     * owner during boot time - This allows the template to be shared when logged in as dummyUser -
+     * Allows this PR to be hero'ed without the need for CRUD functionality
+     */
+    Subject userSubject = SECURITY.getSubject("dummyUser", "dummyPassword");
```

Comment:
Add comment to remove/modify this once CRUD is supported

Reply:
https://github.com/codice/ddf/pull/3105/files/ded0ffdc29b76050353d0c39545c917ab8e16b45#diff-884cd619f7db42eaba207896e9fff009R318